libs/blocks/how-to/how-to.js

```diff
@@ -133,8 +133,18 @@ export default function init(el) {
   }
 
   if (mainVideo) {
-    const videoClass = `how-to-media${isLargeMedia ? ' how-to-media-large' : ''}`;
+    const videoClass = `how-to-media${isLargeMedia ? ' how-to-media-large video' : ''}`;
     el.append(createTag('div', { class: videoClass }, mainVideo));
+    const src = mainVideo.getAttribute('data-video-source');
+    if (src) {
+      createIntersectionObserver({
+        el: mainVideo,
+        options: { rootMargin: '1000px' },
+        callback: () => {
+          mainVideo?.appendChild(createTag('source', { src, type: 'video/mp4' }));
+        },
+      });
```

Comment:
For this part, its better to adapt the intersectionObserver to actually put the intersection observer on the video, rather than its parent element.
https://github.com/adobecom/milo/blob/stage/libs/utils/decorate.js#L334-L336
```
  const videoEl = parentElement.querySelector('video');
  createIntersectionObserver({
    el: videoEl,
```

then you don't need the code here and duplicate it.
The duplication lead to issues previously before